docs/remote_inference_blueprints/amazon_comprehend_connector_blueprint.md

```diff
@@ -0,0 +1,668 @@
+# Amazon Comprehend connector blueprint example for metadata embedding:
+
+Amazon Comprehend uses natural language processing (NLP) to extract insights about the content of documents without the need of any special preprocessing. You can get details of Amazon Comprehend from [Amazon Comprehend Documentation](https://docs.aws.amazon.com/comprehend/).
+
+## Language detection with [DetectDominantLanguage](https://docs.aws.amazon.com/comprehend/latest/APIReference/API_DetectDominantLanguage.html) API
+
+This instruction shows how to create OpenSearch connector having capability to set detected language code based on input text using Amazon Comrehend DetectDominantLanguage API.
+
+Note: Need to do this on 2.14.0 or later.
+
+### 1. Add connector endpoint to trusted URLs:
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+            "^https://comprehend\\..*[a-z0-9-]\\.amazonaws\\.com/.*$"
+        ]
+    }
+}
+```
+
+### 2. Create connector for Amazon Bedrock:
+
+If you are using self-managed Opensearch, you should supply AWS credentials:
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "Amazon Comprehend",
+  "description": "Test connector for Amazon Comprehend",
+  "version": 1,
+  "protocol": "aws_sigv4",
+  "credential": {
+    "access_key": "<PLEASE ADD YOUR AWS ACCESS KEY HERE>",
+    "secret_key": "<PLEASE ADD YOUR AWS SECRET KEY HERE>",
+    "session_token": "<PLEASE ADD YOUR AWS SECURITY TOKEN HERE>"
+  },
+  "parameters": {
+    "service_name": "comprehend",
+    "region": "ap-northeast-1",
+    "endpoint": "https://${parameters.service_name}.${parameters.region}.amazonaws.com",
+    "api": "Comprehend_20171127.DetectDominantLanguage",
+    "response_filter": "$"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "url": "${parameters.endpoint}",
+      "headers": {
+        "X-Amz-Target": "${parameters.api}",
+        "content-type": "application/x-amz-json-1.1",
+      },
+      "request_body": "{ \"Text\": \"${parameters.Text}\"}"
+    }
+  ]
+}
+```
+
+If using the AWS Opensearch Service, you can provide an IAM role arn that allows access to the bedrock service.
+Refer to this [AWS doc](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ml-amazon-connector.html)
```

Comment:
Can you also add suggestion to refer to this demo notebook https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/aws/AIConnectorHelper.ipynb

Reply:
Adding demo a notebook and tutorial guide sounds good. I'm going to work on this.

Reply:
I've added suggestions to refer demo a notebook and tutorial guide